### PR TITLE
GUVNOR-2794: Test firing rules over unique hit policy gdst

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by jomarko** [comment]
Please DO NOT MERGE yet. I am working on an additional test.

**Comment by manstis** [comment]
+1 @jomarko We really do need this type of test. Thank-you!

**Comment by jomarko** [comment]
@manstis no problem. I am done with this PR and moving to another hit policies. You can expect similar PR's next days.

File: drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/GuidedDecisionTableTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package org.drools.testcoverage.functional;
```
Comment (manstis) [inline drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/GuidedDecisionTableTest.java:17]: License

Comment (jomarko) [reply drools-test-coverage/test-suite/src/test/java/org/drools/testcoverage/functional/GuidedDecisionTableTest.java:17]: @manstis thanks. License added.

**Comment by jomarko** [comment]
jenkins please retest this

**Review by manstis — APPROVED** [review]
Thanks @jomarko

**Pull request merged**